autoload/vundle/installer.vim

```diff
@@ -393,6 +393,7 @@ func! s:make_sync_command(bang, bundle) abort
                   \ 'git fetch',
                   \ 'git reset --hard origin/HEAD',
```

Comment:
That would probably work too. In what way is line 396 wrong? I've been using this fork for a while without any issues